frontend/common/src/premium/index.ts

```diff
@@ -126,7 +127,8 @@ export interface SettingsApi {
   update(settings: FrontendSettingsPayload): Promise<void>;
   defaultThemes(): Themes;
   themes(): Themes;
-  user: UserSettingsApi
+  user: UserSettingsApi;
+  i18n: VueI18n;
```

Comment:
I would say instead of exposing the module type, we can just expose a wrapper type.

```Typescript
type I18n = {
 t(params): string
 tc(params): string
}
```

Reply:
uh okay